src/pyOpenMS/pyopenms/addons/featuremap.py

```
@@ -149,8 +149,72 @@ def get_assigned_peptide_identifications(self):
+                for i, fid in enumerate(fids):
+                    if fid is not None and fid not in seen:
+                        seen.add(fid)
+                        keep.append(i)
+                best = best.take(keep)
+                # Build lookup: feature_id -> (sequence, score)
+                lookup = {}
+                for fid, seq, score in zip(
+                    best.column('feature_unique_id').to_pylist(),
+                    best.column('peptidoform').to_pylist(),
+                    best.column('score').to_pylist(),
+                ):
```

Ruff (0.15.6)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)